src/tests/models/fileManager.js

```diff
@@ -6,6 +6,7 @@ const database = require('./database'),
 
 module.exports = {
     createFileFromUrl,
+    downloadFile,
```

Comment:
maybe change
        const errMsg = 'Error to read file, throw exception: ' + err;
to
        const errMsg = 'Error to download file: ' + err;